source/standards/configuration-management.html.md.erb

```diff
@@ -20,6 +20,27 @@ Use [Terraform][] to configure third party cloud infrastructure like Amazon Web
 
 Terraform supports a [large number of providers][], and you can configure it to support multiple environments with different parameters. See the [govuk-aws][] repository as an example.
 
+### Versioning
+
+Due to the high rate of change in many cloud provider offerings we
+recommend you keep your terraform versions and codebases up to date. A
+version manager such as [tfenv](https://github.com/tfutils/tfenv),
+already used by a number of GDS teams, can help you with supporting
+multiple versions.
```

Comment:
Is it worth mentioning `.terraform-version` files here too?

Reply:
That's an implementation detail. Once they look at the tool they'll see it